.github/workflows/gen_json.yml

```diff
@@ -0,0 +1,46 @@
+name: Test API JSON generator
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
```

Comment:
Amazing idea!

Try

```suggestion
on:
  push:
  pull_request:
```

Reply:
go ahead and change it

Reply:
The master needs to be there because the generator is not available on other branches.

Reply:
we can always change that later.

I will adjust it.

Reply:
this line also might be causing an issue..

```yml
    if: github.repository == 'lvgl/lvgl'

```

Reply:
still not showing up in...

Reply:
dammit I keep on forgetting to tag you. @liamHowatt  If I don't do that you don't get a notification